conf/test_kofamscan.config

```
@@ -0,0 +1,29 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/metatdenovo -profile test_eggnog,<docker/singularity>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test eggnog profile'
+    config_profile_description = 'Minimal test dataset to check pipeline with eggnog function added'
```

```suggestion
    config_profile_name        = 'Test kofamscan profile'
    config_profile_description = 'Minimal test dataset to check pipeline with kofamscan function added'
```
This should be kofamscan right?